### DISCUSSION TITLE
Accessibility attributes

### DISCUSSION THREAD

**Discussion by mitera** [ghiscoding/multiple-select-vanilla#178] [Ideas]
Please add accessibility attributes like aria-expanded="false", aria-expanded="true" when the dropdown is displayed.

**Comment by ghiscoding** [comment]
~I added plenty of Aria stuff~, if that is not enough then please **Contribute** a Pull Request since I don't intend to do more and I don't quite understand all options of the Aria topic

oops that was another project that I added many Aria, but regardless please contribute

**Reply by ghiscoding** [reply]
that should be fixed now in v1.1.1
